include/aspect/postprocess/tracer.h

```diff
@@ -35,36 +38,125 @@ namespace aspect
     template <int dim>
     class PassiveTracers : public Interface<dim>, public ::aspect::SimulatorAccess<dim>
     {
+      public:
+        /**
+         * Constructor.
+         */
+        PassiveTracers();
+
+        /**
+         * Destructor.
+         */
+        ~PassiveTracers();
+
+        /**
+         *
+         */
+        void
+        initialize();
+
+        /**
+         * Returns a reference to the particle world, in case anyone wants to
+         * query something about tracers.
+         */
+        const Particle::World<dim> &
+        get_particle_world() const;
+
+        /**
+         * Returns a reference to the particle world, in case anyone wants to
+         * query something about tracers.
+         */
+        Particle::World<dim> &
+        get_particle_world();
```

Comment:
Do you need the non-const version? If so, please mention that this is the non-const version of the function.